.stylelintrc.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
you've added config file, but why didn't you include actual dependency in the MR?